src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -254,12 +257,16 @@ export function isNeverType(type: Type): type is NeverType {
     return type.kind === "Intrinsic" && type.name === "never";
 }
 
+export function isEnumMember(type: Type): boolean {
+    return type.kind === "EnumMember";
+}
+
 export function getInputType(
     program: Program,
     type: Type,
     models: Map<string, InputModelType>,
     enums: Map<string, InputEnumType>
-): InputType {
+): any {
```

Comment:
Why the return value is changed to any. Seems not a good practice.

Reply:
Yes. But this function will be used in callback of `navigateTypesInNamespace` which requires the signature of callback can return other types.

Reply:
Well we should find another way when putting this into the callback instead of compromising our own APIs.

Reply:
emmm...... Can we wrap this method and in that wrapper method we could make it return any?